docusaurus.config.js

```diff
@@ -89,21 +100,77 @@ module.exports = {
           label: 'Networks',
           to: 'networks/iota-1.5-chrysalis',
           activeBaseRegex: '^/networks/.*',
+          type: 'dropdown',
```

Comment:
Just a side note:
We really should try to use Chrysalis and Stardust less. Many people may not even know what it is and I think we don't explain it somewhere 🤔